protocol/synthetix/contracts/mocks/CollateralMockWithoutDecimals.sol

```diff
@@ -0,0 +1,180 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+contract CollateralMockWithoutDecimals {
```

Comment:
probably could have just modified the other mock tocken to optionally revert on the `decimals` call

Reply:
rather than creating a whole new mock